autotest/gdrivers/netcdf.py

```diff
@@ -1107,6 +1107,38 @@ def test_netcdf_27():
     gdal.SetConfigOption("GDAL_NETCDF_BOTTOMUP", config_bak)
 
 
+###############################################################################
+# check support for GDAL_NETCDF_ASSUME_LONGLAT configuration option
+
+
+def test_netcdf_assume_longlat():
+
+    # test default config
+    with gdaltest.config_option("GDAL_NETCDF_ASSUME_LONGLAT", "YES"):
+        ds = gdal.Open("data/netcdf/trmm-nc2.nc")
+        srs = ds.GetSpatialRef()
+        assert srs is not None
+        assert (
+            srs.ExportToWkt()
+            == 'GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Longitude",EAST],AXIS["Latitude",NORTH]]'
+        )
```

Comment:
tiny bit too PROJ version dependent:
```suggestion
        assert srs.ExportToWkt().startswith('GEOGCS["WGS 84')
```